config/config-observability.yaml

```diff
@@ -40,21 +40,44 @@ data:
     # metrics.backend-destination field specifies the system metrics destination.
     # It supports either prometheus (the default) or stackdriver.
     # Note: Using Stackdriver will incur additional charges.
-    metrics.backend-destination: prometheus
 
     # metrics.stackdriver-project-id field specifies the Stackdriver project ID. This
     # field is optional. When running on GCE, application default credentials will be
     # used and metrics will be sent to the cluster's project if this field is
     # not provided.
-    metrics.stackdriver-project-id: "<your stackdriver project id>"
 
     # metrics.allow-stackdriver-custom-metrics indicates whether it is allowed
     # to send metrics to Stackdriver using "global" resource type and custom
     # metric type. Setting this flag to "true" could cause extra Stackdriver
     # charge.  If metrics.backend-destination is not Stackdriver, this is
     # ignored.
+
+    # logging.enable-var-log-collection specifies whether the logs under /var/log/ should be available
+    # for collection on the host node by the fluentd daemon set.
+
+    # logging.revision-url-template is a string containing the logging url template where
+    # the variable REVISION_UID will be replaced with the created revision's UID.
+
+    # logging.request-log-template is the go template to use to shape the request logs.
+
+    # logging.enable-request-log enables activator/queue-proxy to write request logs.
+
+    # profiling.enable indicates whether it is allowed to retrieve runtime profiling data from
+    # the pods via an HTTP server in the format expected by the pprof visualization tool.
+
+    # metrics.opencensus-address specifies the metrics collector address. This is only used
+    # when the metrics backend is opencensus.
+
     metrics.allow-stackdriver-custom-metrics: "false"
     metrics.taskrun.level: "task"
     metrics.taskrun.duration-type: "histogram"
     metrics.pipelinerun.level: "pipeline"
     metrics.pipelinerun.duration-type: "histogram"
+    logging.enable-var-log-collection: "false"
+    logging.revision-url-template: ""
+    logging.request-log-template: '{"httpRequest": {"requestMethod": "{{.Request.Method}}", "requestUrl": "{{js .Request.RequestURI}}", "requestSize": "{{.Request.ContentLength}}", "status": {{.Response.Code}}, "responseSize": "{{.Response.Size}}", "userAgent": "{{js .Request.UserAgent}}", "remoteIp": "{{js .Request.RemoteAddr}}", "serverIp": "{{.Revision.PodIP}}", "referer": "{{js .Request.Referer}}", "latency": "{{.Response.Latency}}s", "protocol": "{{.Request.Proto}}"}, "traceId": "{{index .Request.Header "X-B3-Traceid"}}"}'
+    logging.enable-probe-request-log: "false"
+    metrics.request-metrics-backend-destination: "prometheus"
+    metrics.stackdriver-project-id: "<your stackdriver project id>"
+    profiling.enable: "false"
+    metrics.opencensus-address: ""
```

Comment:
Shouldn't these be commented by default. I think it's good to have them to "document" the optional parameters, but as is, they could make things not work (like `<your stackdriver project id>`) when using apply the release.yaml